desktop/apps/auction/components/layout/Banner.js

```diff
@@ -16,6 +17,16 @@ function Banner (props) {
     name
   } = props
 
+  const trackEnterLive = () => {
```

Comment:
👍  LGTM. There's probably a discussion to be had about using [React Tracking](https://github.com/NYTimes/react-tracking) vs. raw analytics calls, b/c that's what we use in Reaction—for a separate time though, not going to hold up this PR.